examples/00_quick_start/sasrec-tf2.ipynb

```diff
@@ -0,0 +1,321 @@
+{
```

Comment:
<p>can you confirm that this guy works in a fresh run? Inside this code is:</p><pre class="ql-syntax" spellcheck="false">user_vocab : ./tests/resources/deeprec/slirec/user_vocab.pkl
</pre><p>so it looks you may need to download these files in advance. See download and load data in https://github.com/microsoft/recommenders/blob/main/examples/00_quick_start/lstur_MIND.ipynb</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1530/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>This was copied from slirec. I am not using the config file. Removed the folder. </p>